Mongle/Projects/Features/OnBoardingFeature/Sources/Views/PhoneNumberVerification/PhoneNumberVerificationView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  PhoneNumberVerificationView.swift
+//  OnBoardingFeature
+//
+//  Created by bulmang on 4/15/24.
+//  Copyright © 2024 Mongle-iOS. All rights reserved.
+//
+
+import SwiftUI
+import UI
+import Core
+
+public struct PhoneNumberVerificationView: View {
+    @Environment(\.presentationMode) var presentationMode
+    @StateObject private var viewModel: PhoneNumberVerificationViewModel
+    @State private var isRequestVerifyNumber = false
+    @State private var phoneNumber: String = ""
+    @State private var verifyCode: String = ""
+
+    public init(viewModel: PhoneNumberVerificationViewModel) {
+        _viewModel = StateObject(wrappedValue: viewModel)
+    }
```

Comment:
```suggestion
    public init() {
        _viewModel = StateObject(wrappedValue: PhoneNumberVerificationViewModel(otpRepository: OTPRepositoryImp(baseURL: ConfigImp().baseURL.toURL(), network: NetworkImp())))
    }
 
```
초기화할때 주입하는 것은 어떻게 생각하시나요?